ansible/ocemr/tasks/system_config.yml

```diff
@@ -0,0 +1,51 @@
+---
+- hostname:
+    name: "{{ base.net_hostname }}"
+
+- replace:
+    path: /etc/hosts
+    regexp: '^127.0.1.1.*'
+    replace: '127.0.1.1\t{{ base.net_hostname }}.{{ base.net_domain }}\t{{ base.net_hostname }}'
+    backup: yes
+
+- name: set timezone
+  timezone:
+    name: "{{ base.timezone }}"
+
+- name: configure dhcp networking
+  template:
+    src: network_interfaces.dhcp.j2
+    dest: /etc/network/interfaces
+    owner: root
+    group: root
+  when: base.net_dhcp
+  notify:
+    - Restart Networking
+
+- name: configure static networking
+  template:
+    src: network_interfaces.static.j2
+    dest: /etc/network/interfaces
+    owner: root
+    group: root
+  when: not base.net_dhcp
+  notify:
+    - Restart Networking
+
+- name: configure resolv.conf
+  template:
+    src: resolv_conf.j2
+    dest: /etc/resolv.conf
+    owner: root
+    group: root
+  when: not base.net_dhcp
+
+- name: enable serial console
+  systemd:
+    name: serial-getty@ttyS0.service
+    enabled: True
+
+- name: start serial console
+  systemd:
+    name: serial-getty@ttyS0.service
+    state: started
```

Comment:
This task hangs indefinetly in my KVM test environment..

Reply:
Might be the serial console part. We can add a variable around it, but I'm unsure how to flip it based on the system installed to.

Reply:
Building for an appliance, not a kvm.

Reply:
Damn chickens! serial-getty must be run after rc.local, but was being called from rc.local. Fixed in the installer: https://github.com/ph1l/ocemr_installer/pull/7/files#diff-8623f738c4ee33ec558f04a28da41399R12